core/templates/services/exploration-features.service.ts

```diff
@@ -73,18 +83,30 @@ export class ExplorationFeaturesService {
     ExplorationFeaturesService.serviceIsInitialized = true;
   }
 
+  /**
+   * @returns - Whether a service is initialized or not.
```

Comment:
When you start with whether, you don't need to add an "or not" at the end.

Ditto below.

Reply:
Sure! Will keep that in mind. Updated.